src/views/Connect.vue

```diff
@@ -4,7 +4,7 @@
     :onSuccess="onSuccess"
     :contextName="contextName"
     :logo="logo"
-    :loginText="loginText"
+    :loginText="contextName"
```

Comment:
Not sure I understand. What is `loginText`? Is it displayed to the user? If so, the context name is not appropriate.

Reply:
Yes, the `loginText` will be displayed to the user when using the `vda-login` plugin to log in, it is a custom call to action button label to log in with verida connect. 

however, The `loginText` props name is not really appropriate will change it to `Button Label` (later when upgrading the component library)

Reply:
So why choosing to display the context name to the user rather than a user-friendly and more relevant text?

Reply:
youre right, I will add the  login text env back

Reply:
updated